deskStructure.js

```diff
@@ -1,8 +1,9 @@
 import * as Structure from 'sanity-plugin-intl-input/lib/structure';
+import S from '@sanity/desk-tool/structure-builder';
 
 // default implementation by re-exporting
-export const getDefaultDocumentNode = Structure.getDefaultDocumentNode;
-export default Structure.default;
+// export const getDefaultDocumentNode = Structure.getDefaultDocumentNode;
```

Comment:
Keeping these commented lines intentionally?

Reply:
yes keeping for later when we add i18n support (this is config related to the sanity i18n plugin)